nemo/collections/asr/data/audio_to_label.py

```diff
@@ -206,6 +209,53 @@ def sliced_seq_collate_fn(self, batch):
 
         return audio_signal, audio_lengths, tokens, tokens_lengths
 
+    def vad_frame_seq_collate_fn(self, batch):
+        """collate batch of audio sig, audio len, tokens, tokens len
+        Args:
+            batch (Optional[FloatTensor], Optional[LongTensor], LongTensor,
+                LongTensor):  A tuple of tuples of signal, signal lengths,
+                encoded tokens, and encoded tokens length.  This collate func
+                assumes the signals are 1d torch tensors (i.e. mono audio).
+                batch size equals to 1.      
+        """
+        slice_length = int(self.featurizer.sample_rate * self.time_length)
+        _, audio_lengths, _, tokens_lengths = zip(*batch)
+        slice_length = min(slice_length, max(audio_lengths))
```

Comment:
why is this required to check for min and max for audio_lengths and slice_length?

Reply:
no it's not required. It's just I worried that user would input audio length really less than sample length. 🤣